.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,78 @@
+name: Build & Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  scalafmt:
+    name: Check formatting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache SBT ivy cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.ivy2/cache
+          key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}
```

Comment:
Uh, I noticed that in the .circleci/config.yml we also only reference `build.sbt`. But the cache can also change if `project/plugins.sbt`, `project/build.properties` or the CI-definition change (*). We should include them!

----
(*) You will notice that when you change the cache-key, and it'll continue to use the old cache, because the CI-definition wasn't part of the cache key so far. Good Luck! 🍀

Reply:
Improved caching by combining it into one step including `plugins.sbt` and `build.properties'.
I just tried it out. GitHub always uses the updated cache-key. It basically does restoring and caching in one action (with hooks).

Reply:
👍